qt/scientific_interfaces/ISISReflectometry/Common/ModelCreationHelper.cpp

```diff
@@ -0,0 +1,316 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#include "ModelCreationHelper.h"
+#include "../../ISISReflectometry/Reduction/Batch.h"
+
+namespace MantidQt {
+namespace CustomInterfaces {
+namespace ModelCreationHelper {
+
+namespace { // unnamed
+Row makeRowWithOutputNames(std::vector<std::string> const &outputNames) {
+  auto row = Row({}, 0.0, TransmissionRunPair(), RangeInQ(), boost::none,
+                 ReductionOptionsMap(),
+                 ReductionWorkspaces({}, TransmissionRunPair()));
+  row.setOutputNames(outputNames);
+  return row;
+}
+} // namespace
+
+/* Rows */
+
+Row makeEmptyRow() {
+  return Row({}, 0.0, TransmissionRunPair(), RangeInQ(), boost::none,
+             ReductionOptionsMap(),
+             ReductionWorkspaces({}, TransmissionRunPair()));
+}
+
+Row makeRow(double theta) {
+  return Row({}, theta, TransmissionRunPair({"22348", "22349"}), RangeInQ(),
```

Comment:
Intentional, but this file should only be used for testing. It should perhaps be in a different location to make that clear but I'm not sure where. I guess it would be better to have a separate `TestHelpers` subdirectory rather than putting it in `Common`?

Reply:
I agree, it's not initially obvious this is for testing only.

Moving this can be done later though.

Reply:
I've added a note on the parent issue to tidy this up later